rust/Cargo.toml

```diff
@@ -19,6 +18,11 @@ serde = { version = "1.0.138", features = ["derive"] }
 inventory = "0.3.0"
 walkdir = "2.3.2"
 
+[dependencies.libc]
+git = "https://github.com/rust-lang/libc"
+# TODO: Set version with new makedev platforms
```

Comment:
What does this TODO mean?

Reply:
Just to change the version when a new libc version will be released with the makedev functions.